### PR TITLE
New Alias: Epsilon (Conversant)

### PR DESCRIPTION
**Pull request by AaronColbyPrice**
Updating some branding across all projects from Conversant to Epsilon

### CODE REVIEW

**Comment by onkarvhanumante** [comment]
@AaronColbyPrice,  `Eps branding`  implies that you want to new alias epsilon in conversant adapter? Could you confirm

File: static/bidder-info/conversant.yaml
Comment (onkarvhanumante) [inline static/bidder-info/conversant.yaml:3]: @AaronColbyPrice  
Thank you for your contribution to the Prebid server. An email is sent from prebid team to verify `PublisherIntegration@epsilon.com` email address. Please respond back with "received" message to complete verification

Comment (onkarvhanumante) [reply static/bidder-info/conversant.yaml:3]: @AaronColbyPrice  email has bounced back with following message. Please fix it. 

```
Your message to PublisherIntegration@epsilon.com couldn't be delivered.
The group PublisherIntegration only accepts messages from people in its organization or on its allowed senders list, and your email address isn't on the list.
```

Comment (AaronColbyPrice) [reply static/bidder-info/conversant.yaml:3]: @onkarvhanumante excellent catch, we are testing now and it works internally, but not externally.  We are fixing now and I'll follow up.

File: static/bidder-info/conversant.yaml
Comment (onkarvhanumante) [inline static/bidder-info/conversant.yaml:18]: this info is related to `conversant`. Any specific reason to change it to `epsilon`

File: static/bidder-params/conversant.json
Comment (onkarvhanumante) [inline static/bidder-params/conversant.json:9]: this info is related to conversant. Any specific reason to mention  epsilon here

Comment (AaronColbyPrice) [reply static/bidder-params/conversant.json:9]: Conversant Media doesn't exist anymore, we've merged with Epsilon so this works is to try and start moving our branding in that direction.  However to avoid confusion, it is good to reference our legacy name until we are 100% sure we want to remove it completely.

**Comment by AaronColbyPrice** [comment]
@onkarvhanumante regarding comment:
@AaronColbyPrice, Eps branding implies that you want to new alias epsilon in conversant adapter? or do you want to deprecate conversant and use epsilon. Could you confirm?

Eventually the goal is to deprecate conversant.  The first step is to add in an alias and encourage new publishers to use that alias without breaking anything for existing publishers.  Our development team is very cautious so this is a stepping stone to that goal so both statements are correct.  We want Epsilon as an alias and eventually want to deprecate conversant but we're just not there yet.

**Comment by onkarvhanumante** [comment]
> @onkarvhanumante regarding comment: @AaronColbyPrice, Eps branding implies that you want to new alias epsilon in conversant adapter? or do you want to deprecate conversant and use epsilon. Could you confirm?
> 
> Eventually the goal is to deprecate conversant. The first step is to add in an alias and encourage new publishers to use that alias without breaking anything for existing publishers. Our development team is very cautious so this is a stepping stone to that goal so both statements are correct. We want Epsilon as an alias and eventually want to deprecate conversant but we're just not there yet.

Thanks @AaronColbyPrice  for the clarification.

**Comment by Sonali-More-Xandr** [comment]
Ready to approve once the email verification step is completed!

File: static/bidder-info/conversant.yaml
Comment (onkarvhanumante) [reply static/bidder-info/conversant.yaml:3]: @AaronColbyPrice let us know when to verifiy email again

**Comment by Sonali-More-Xandr** [comment]
@AaronColbyPrice Any updates on the maintainer email address?

**Comment by AaronColbyPrice** [comment]
> @AaronColbyPrice Any updates on the maintainer email address?

Sorry was on vacation, thanks for your patience, we believe it is working now so are ready to try to validate again

**Comment by Sonali-More-Xandr** [comment]
> > @AaronColbyPrice Any updates on the maintainer email address?
> 
> Sorry was on vacation, thanks for your patience, we believe it is working now so are ready to try to validate again

Thank you, @AaronColbyPrice! We have sent an email from Prebid team to verify `PublisherIntegration@epsilon.com` email address. Please respond back with "received" message to complete verification

**Comment by Sonali-More-Xandr** [comment]
> > > @AaronColbyPrice Any updates on the maintainer email address?
> > 
> > 
> > Sorry was on vacation, thanks for your patience, we believe it is working now so are ready to try to validate again
> 
> Thank you, @AaronColbyPrice! We have sent an email from Prebid team to verify `PublisherIntegration@epsilon.com` email address. Please respond back with "received" message to complete verification

Email verification completed!

File: static/bidder-info/conversant.yaml
Comment (Sonali-More-Xandr) [reply static/bidder-info/conversant.yaml:3]: Email verification is completed!

**Review by Sonali-More-Xandr — APPROVED** [review]

**Review by onkarvhanumante — APPROVED** [review]

**Pull request merged**
